src/index.html

```diff
@@ -11,37 +11,32 @@
   </head>
   <body>
     <header class="header" role='banner'>
-      <div class="switch__dropdown">
-        <form class="switch">
-          <input name="switch" id="dark" type="radio" value="Dark" checked />
-          <label for="dark" class="switch__label">Dark</label>
-          <input name="switch" id="light" type="radio" value="light" />
-          <label for="light" class="switch__label">Light</label>
-          <div class="switch__indicator"></div>
-        </form>
-        <section class="dropdown">
-          <input type="hidden" value="All info">
-          <header><p>All info</p></header>
-          <div>
-            <p>All info</p>
-            <p>Today</p>
-            <p>Week</p>
-            <p>Average</p>
-          </div>
-        </section>
-      </div>
+      <form class="toggle">
+      	<input type="checkbox" id="toggle-switch" />
+      	<label for="toggle-switch"><span class="screen-reader-text">Toggle Color Scheme</span></label>
```

Comment:
Is this span something we could change to a button for accessibility?

Reply:
That's a good point @Garrett-Iannuzzi, something we should consider. If we lose some of the aesthetics by switching to a button we could probably add an event listener to the span that fires a onKeyDown event, and only fires if the keycode === 13. Should circle back to this as we design the input forms functionality/accessibility for POST network requests